restore.go

```diff
@@ -50,14 +50,9 @@ var restoreCommand = cli.Command{
 			Usage: "cgroups mode: 'soft' (default), 'full' and 'strict'.",
 		},
 		cli.StringFlag{
-			Name:  "config-file, c",
-			Value: "config.json",
-			Usage: "path to spec file for writing",
-		},
-		cli.StringFlag{
-			Name:  "runtime-file, r",
-			Value: "runtime.json",
-			Usage: "path for runtime file for writing",
+			Name:  "bundle-dir, b",
```

Comment:
`-b` is the short name :-) 
Pros & Cons of `--bundle-dir` vs `--bundle`... Is that what it really is or is it just the root directory / location of the bundle?  If the term bundle is to be defined as a directory containing the following... Then it makes sense to use `--bundle`   If term bundle is to be defined more as the contents of the said directory.. then this should probably be called bundle-dir or bundle-path or bundle-root.  

Thinking out loud,... if it is the bundle itself then one might want to say if it's a directory it contains... and if it's a path to an iso, rar, or tar then we mount it and process accordingly.